src/open_dread_rando/door_locks/custom_door_types.py

```diff
@@ -199,8 +203,8 @@ class ShieldData:
     "bomb": ShieldData(
         name="doorshieldbomb",
         type=DoorTemplates.TRIANGLES,
-        weaknesses=[], # "BOMB" weakness is added in runtime lua.
-                       # see open_dread_rando/files/custom_scenario.lua : Scenario._BlastShieldTypes
```

Comment:
This space was there to make it clear that it's a continuation of the above comment. I'd either keep it the same or move it all to a comment above `weaknesses = []`.